src/components/EventSearchInput/EventSearchInput.sass

```diff
@@ -1,2 +1,12 @@
 .input-search-wrapper
   display: block
+  position: relative
+
+  input
+    padding: 8px 20px 8px 34px
+
+  svg
+    position: absolute
+    fill: #3f474d
```

Comment:
instead of hardcoding the color, you should import the variables at the top and using `$main-dark-gray`